crates/sui-graphql-rpc/src/mutations.rs

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::error::code;
+
+use crate::{error::graphql_error, types::transaction_exec::ExecutionResult};
+use async_graphql::*;
+use fastcrypto::{encoding::Base64, traits::ToFromBytes};
+use shared_crypto::intent::Intent;
+use sui_json_rpc_types::SuiTransactionBlockResponseOptions;
+use sui_sdk::SuiClient;
+use sui_types::quorum_driver_types::ExecuteTransactionRequestType;
+use sui_types::{signature::GenericSignature, transaction::Transaction};
+
+pub struct Mutation;
+
+#[Object]
+impl Mutation {
+    async fn execute_transaction_block(
+        &self,
+        ctx: &Context<'_>,
+        tx_bytes: String,
+        signatures: Vec<String>,
+    ) -> Result<ExecutionResult> {
+        let sui_sdk_client: &SuiClient = ctx.data().map_err(|_| {
+            graphql_error(
+                code::INTERNAL_SERVER_ERROR,
+                "Unable to fetch Sui SDK client",
+            )
+        })?;
+
+        let tx_data = bcs::from_bytes(
+            &Base64::try_from(tx_bytes)
+                .map_err(|e| {
+                    graphql_error(
+                        code::INTERNAL_SERVER_ERROR,
+                        format!("Unable to deserialize transaction to Base64: {:?}", e),
+                    )
+                })?
+                .to_vec()
+                .map_err(|e| {
+                    graphql_error(
+                        code::INTERNAL_SERVER_ERROR,
+                        format!("Unable to decode Base64 to vec: {:?}", e),
+                    )
+                })?,
+        )?;
```

Comment:
This pattern decodes each Base64 value twice -- once during `try_from` and once in `to_vec`.  Instead  we can use `Base64::decode` (part of the `Encoding` trait) to do it once and directly get a vector of bytes.

Also, the errors here are written with reference to how the code is written, rather than from the perspective of the person reading the error.  Can we reword from that perspective? Finally, we should use the `Display` format for the error, rather than the `Debug` format.

```suggestion
        let tx_data = bcs::from_bytes(
            &Base64::decode(&tx_bytes)
                .map_err(|e| {
                    graphql_error(
                        code::INTERNAL_SERVER_ERROR,
                        format!("Unable to deserialize transaction from Base64: {e}"),
                    )
                });
        )?;
```

Similar feedback applies in all the other places where we're doing Base64 deserialization -- maybe it's worth factoring that out into a helper function.